import AsyncError from "../Errors/AsyncError";
import user from "../Models/users";

export const login = AsyncError((req, res, next) => {});
